tree/dataframe/src/RDataFrame.cxx

```diff
@@ -629,6 +629,31 @@ df = RDataFrame("mytree", "myfile.root", sparkcontext = sc)
 If an instance of [SparkContext](https://spark.apache.org/docs/latest/api/python/reference/api/pyspark.SparkContext.html)
 is not provided, the default behaviour is to create one in the background for you.
 
+### Distributed Snapshot
+
+The Snapshot operation behaves slightly differently when executed distributedly. First off, it requires the path
+supplied to the Snapshot call to be accessible from any worker of the cluster. Then, each distributed task will make
+a Snapshot on the range of entries it has been assigned with by the distributed RDataFrame application. Let's suppose
+our dataset is made of 100 entries. The default number of partitions in which the data will be split for distributed
+execution is 2. This means that a Snapshot operation on this dataset will produce two separate files, each holding the
+partial dataset resulting from applying the RDataFrame computation graph on either of the two ranges of entries.
+For example, the following application:
+
+~~~{.py}
+import ROOT
+RDataFrame = ROOT.RDF.Experimental.Distributed.Spark.RDataFrame
+df = RDataFrame(100)
+# Define a single column of 100 consecutive numbers
+df_x = df.Define("x","rdfentry_")
+# Snapshot the dataset distributedly
+snapdf = df_x.Snapshot("mytree","myfile.root")
+~~~
+
+Will result in two snapshotted files, respectively `myfile_0_49.root` and `myfile_50_99.root` where the numbers appended
+to the file name represent the initial and final entry of the range processed by each distributed task. As with local
+RDataFrame, the result of a Snapshot on a distributed RDataFrame is another distributed RDataFrame on which we can define
+a new computation graph and run more distributed computations.
```

Comment:
I think the example should refer to the general case, not a single-machine run. if I understand correctly using a relative path breaks subsequent usage of the returned RDF (in the general case), because it won't be able to access the input files which are stored on the workers' local storage. if that's the case distRDF should error out or at least warn if a relative path is provided?